docs/_docs/bestpractices/sustainability/sustainability.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+parent: Best practices
+permalink: /bestpractices/Sustainability
+nav_order: 2
+title: Sustainability
+author: arclares
+ms.date: 08/16/2024
+ms.service: finops
+description: 'Discover essential FinOps best practices to optimize cost efficiency and governance for your Azure resources.'
+
+
+---
+
+<span class="fs-9 d-block mb-4">Sustainability</span>
+Discover essential FinOps best practices to optimize cost efficiency and governance for your Azure resources.
+{: .fs-6 .fw-300 }
+
+[Share feedback](#️-looking-for-more){: .btn .fs-5 .mb-4 .mb-md-0 .mr-4 }
+
+<details open markdown="1">
+   <summary class="fs-2 text-uppercase">On this page</summary>
+
+- [🍃 Carbon Optimization](#carbon-optimization)
+- [🙋‍♀️ Looking for more?](#️-looking-for-more)
+- [🧰 Related tools](#-related-tools)
+
+</details>
+
+---
+
+
+## Carbon Optimization

```

Comment:
What Azure Services does this cover? Is it only VMs or does it cover many services? These services are organized by FOCUS ServiceCategory and Sustainability doesn't fit, which makes me question if there's a better way to categorize this. If not, then I'm thinking it might be better to have a "General" category for anything that applies to all resource types. In general, we want carbon emissions queries for the services to be grouped with the services and not ACO.

Reply:
Good question. This query is only for Carbon Emissions. So far, the only recommendations I've seen are related to VM right size, but I don't think it is limited to VMs only. 
<img width="667" alt="image" src="https://github.com/user-attachments/assets/f608d50d-403e-48a3-9d72-f5e086621b99">

Reply:
It basically says that if you right size your oversized VMs, you will reduce your carbon footprint.